src/_lib/modules/feature-alerts/feature-alerts.module.ts

```diff
@@ -0,0 +1,24 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FeatureAlertComponent } from './feature-alert.component';
+import { FeatureAlertsService } from './feature-alerts.service';
+import { WindowRef } from './windowref';
+import {PopoverModule} from 'ngx-bootstrap';
```

Comment:
Just for uniformity, space after opening bracket and before closing bracket.

Also import from `ngx-bootstrap/popover`

Reply:
Done.